Xamarin.Forms.Core/Color.cs

```diff
@@ -41,46 +41,92 @@ public double A
 			get { return _a; }
 		}
 
-		readonly float _r;
+		readonly float _rh;
```

Comment:
user _component0, _component1, and _component2, or an array

Reply:
I don't think array is a good option because it always allocates memory on heap.

 _component0, _component1, and _component2 are not as good as _rh,  _gs and _bl in my opinion because they are long and they don't explicitly state what may be inside of those field.
May I have the reason about this comment?

Reply:
> I don't think array is a good option because it always allocates memory on heap.

starting with c#7.2, you can use `stackalloc` in an unsafe context, which is totally suitable here

Reply:
for me, _comp0 carries more information than _rh

Reply:
Not agreed, _comp0 carries no information to me. This is kind of like naming _a as _comp4.
But as you wish. :smirk: